src/notify.js

```diff
@@ -59,6 +63,14 @@ function Notify(title, options = {}) {
     }
 }
 
+Notify.serviceWorkerSupported = function() {
```

Comment:
Ok, I see in your case why you want to avoid the push API. I don't agree on it being outside the scope of this library though. The purpose is to make using notifications for users easier, which I think still fits in there. I'll give it some more thought and perhaps follow up.